src/test/java/se/sundsvall/educationfinder/api/CoursesResourceTest.java

```diff
@@ -348,4 +352,35 @@ void findFilterValuesForStudyLocation() {
 			.hasSize(5)
 			.containsExactly("HÄRNÖSAND", "KRAMFORS", "SUNDSVALL", "ÖRNSKÖLDSVIK", "ÖSTERSUND");
 	}
+
+	private static Stream<Arguments> queryParameters() {
```

Comment:
Should probably be in the top of the class.

Reply:
I see your point. I just think its nice to keep static helpers at the top, so they dont get lost. But do as you feel.